src/next-collection-view.js

```diff
@@ -433,7 +438,7 @@ const CollectionView = Backbone.View.extend({
     delete this._addedViews;
 
     if (!viewFilter) {
-      if (addedViews && _.every(addedViews, _.bind(this._isAddedAtEnd, this))) {
+      if (!this.sortWithCollection && addedViews && _.every(addedViews, _.bind(this._isAddedAtEnd, this))) {
```

Comment:
maybe start switching `_.bind` to `this._isAddedAtEnd.bind(this)`?

Reply:
I think to keep consistency we should make stylistic or strategic changes in singular PRs so we don't end up with multiple ways of doing the same task